phpunit/class-gutenberg-rest-templates-controller-test.php

```diff
@@ -1,276 +0,0 @@
-<?php
-
-class Gutenberg_REST_Templates_Controller_Test extends WP_Test_REST_Controller_Testcase {
```

Comment:
This function is not in Gutenberg anymore. cc @ntsekouras @ramonjd  for confirmation. Let's also confirm that the tests were backported properly.

Reply:
The ones I had added have been backported: https://github.com/WordPress/wordpress-develop/pull/3221